pkg/kopia/command/repository.go

```diff
@@ -27,13 +27,12 @@ import (
 // creating or connecting to a Kopia repository
 type RepositoryCommandArgs struct {
 	*CommandArgs
-	CacheDirectory  string
-	Hostname        string
-	ContentCacheMB  int
-	MetadataCacheMB int
-	Username        string
-	RepoPathPrefix  string
-	ReadOnly        bool
+	CacheArgs
```

Comment:
Can we have two implicit variables?
I guess one's a pointer and other's a value so should work okay